SECURITY.md

```diff
@@ -35,3 +35,4 @@ A public disclosure date is negotiated by the Hasura product security team and t
 This document is available in the following translations:
 
 - [French :fr:](translations/SECURITY.french.md)
+- [Hindi :hi:](translations/SECURITY.hindi.md)
```

Comment:
The flag doesn't show here:

![image](https://user-images.githubusercontent.com/5722022/67632274-ba44b980-f8a1-11e9-8102-ec12c3298539.png)